### PR TITLE
chore: (mac) replace deprecated standalone eda-ui image with nginx

### PR DESCRIPTION
**Pull request by zkayyali812**
Mac equivalent PR of - https://github.com/ansible/eda-server/pull/1071

Removes eda-ui in favor of nginx on mac docker-compose

### CODE REVIEW

File: tools/docker/docker-compose-mac.yml [hunk 1]
```diff
@@ -209,13 +210,16 @@ services:
...
       - '${EDA_PROXY_PORT:-3128}:3128'
 
-  eda-ui:
-    image: ${EDA_UI_IMAGE:-quay.io/ansible/eda-ui}:${EDA_UI_VERSION:-main}
+  eda-nginx:
+    image: ${EDA_NGINX_IMAGE:-docker.io/nginx:alpine}
     environment: *common-env
+    command: nginx -g "daemon off;"
     ports:
       - '${EDA_UI_PORT:-8443}:443'
```
Comment (bzwei) [inline tools/docker/docker-compose-mac.yml:218]: should we change EDA_UI_PORT to EDA_API_PORT?

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose-mac.yml:218]: I didn't it in other places to keep backward compatibility, there is other development stuff that currently relies on this envvar.

**Review by Alex-Izquierdo — APPROVED** [review]

File: tools/docker/docker-compose-mac.yml (hunk 1 above)
Comment (zkayyali812) [reply tools/docker/docker-compose-mac.yml:218]: @bzwei exactly what Alejandro is noting. This also maintains consistency with the changes here - https://github.com/ansible/eda-server/pull/1071/files

**Review by bzwei — APPROVED** [review]

File: tools/docker/docker-compose-mac.yml (hunk 1 above)
Comment (bzwei) [reply tools/docker/docker-compose-mac.yml:218]: we can do a later PR that removes other dependencies

**Pull request merged**